From 0a6ef7ee6fe4541386a352881e0834b37835578f Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Tue, 5 Sep 2017 18:02:13 +0000
Subject: [PATCH] OS-6324 Need support for LSI 9305-24i Reviewed by: Dan
 McDonald <danmcd@joyent.com> Approved by: Joshua M. Clulow <jmc@joyent.com>

---
 overlay/generic/etc/driver_aliases | 1 +
 1 file changed, 1 insertion(+)

diff --git a/overlay/generic/etc/driver_aliases b/overlay/generic/etc/driver_aliases
index eda1c7b9..8959edc1 100644
--- a/overlay/generic/etc/driver_aliases
+++ b/overlay/generic/etc/driver_aliases
@@ -920,6 +920,7 @@ mpt_sas "pciex1000,94"
 mpt_sas "pciex1000,95"
 mpt_sas "pciex1000,96"
 mpt_sas "pciex1000,97"
+mpt_sas "pciex1000,c4"
 mr_sas "pciex1000,52"
 mr_sas "pciex1000,53"
 mr_sas "pciex1000,5b"
-- 
2.21.0

